launcher/src/main/java/org/apache/spark/launcher/Main.java

```diff
@@ -101,12 +101,9 @@ public static void main(String[] argsArray) throws Exception {
    * The method quotes all arguments so that spaces are handled as expected. Quotes within arguments
    * are "double quoted" (which is batch for escaping a quote). This page has more details about
    * quoting and other batch script fun stuff: http://ss64.com/nt/syntax-esc.html
-   *
-   * The command is executed using "cmd /c" and formatted in single line, since that's the
-   * easiest way to consume this from a batch script (see spark-class2.cmd).
    */
   private static String prepareWindowsCommand(List<String> cmd, Map<String, String> childEnv) {
-    StringBuilder cmdline = new StringBuilder("cmd /c \"");
+    StringBuilder cmdline = new StringBuilder("");
```

Comment:
I assume you're sure about this, but we definitely don't always need `cmd /c` to run a windows command directly from Java? In Linux you have to have `bash` to interpret most of what we're used to typing on the command line.